lib/fungus_toast/games/active_game.ex

```diff
@@ -0,0 +1,17 @@
+defmodule FungusToast.Games.ActiveGame do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+
+  schema "active_games" do
+
+    timestamps()
+  end
+
+  @doc false
+  def changeset(active_game, attrs) do
+    active_game
+    |> cast(attrs, [])
+    |> validate_required([])
+  end
+end
```

Comment:
@jakejgordon For the time being, I named the `game` structure `ActiveGame` with a corresponding `active_games` table. We can change this - it's just what came to mind after our convo last night. There's nothing on this right now accept [timestamps](https://hexdocs.pm/ecto/Ecto.Schema.html#timestamps/1) and the [default id](https://hexdocs.pm/ecto/Ecto.Schema.html#module-primary-keys) as the primary key.

Reply:
Is there a need to keep game information past the "active" state?  Active games seems like something that could be tracked in ETS or something.  Just sharing my initial reaction to this, I'm sure there's more to the story that I'm unaware of though.

Reply:
I believe there is. Since gameplay could be async, a player may request to play a game, so the game would exist, but it wouldn't be active until other players joined the game. Originally, I named this `ActiveGame` thinking we'd only have 1 active game at a time as we started this. I updated and just made this game. We can iterate on the statefulness part of this. Jake and I were just having a quick convo and thought maybe a bool or timestamp for active. This might be something we don't need now though and could just warrant a migration in the future.

Reply:
> @jakejgordon For the time being, I named the `game` structure `ActiveGame` with a corresponding `active_games` table. We can change this - it's just what came to mind after our convo last night. There's nothing on this right now **_accept_** [timestamps](https://hexdocs.pm/ecto/Ecto.Schema.html#timestamps/1) and the [default id](https://hexdocs.pm/ecto/Ecto.Schema.html#module-primary-keys) as the primary key.

Should be "except", but I'll have to accept your comment as is.

Seriously though, I think having a game_status_id (with FK to a GameStatuses table) is probably a better idea. Then we can include NotStarted, InProgress, Abandoned, RecentlyFinished, and FinishedAndArchived for status. Setting those states would be a function of whether the game started, whether it was finished, and the time the game was last updated.